### PR TITLE
Change the model and the training method

### PR DESCRIPTION
**Pull request by ghost**
- 5*5->3*3(filter size of the last conv. layer)
 - 5*5->7*7(size of output feature map)
 - new training method

### CODE REVIEW

**Comment by ghost** [comment]
@D0048 , you need to review this. I will merge it soon.

File: train_mnist.py [hunk 1]
```diff
@@ -126,29 +126,15 @@ def scscn(x, num, num_conv):
...
-                      [-1,
-                      m,
-                      n,
-                      num * num_conv]), 5, 5), [-1, num_conv]), keep_prob
+            slicing = slicing.write(h,
+                        tf.slice(x, [0, j * stride, k * stride, 0],
+                                    [-1, a, b, -1]))
+    with tf.name_scope('scn'):
+        scn_input = slicing.concat()
+        output = small_cnn(scn_input, num_conv, keep_prob)
```
Comment (D0048) [inline train_mnist.py:134]: This call looks wrong. You may want to fix it.

File: train_mnist.py [hunk 2]
```diff
@@ -126,29 +126,15 @@ def scscn(x, num, num_conv):
             l = int(h % (m*n))
             j = int(l / n)
             k = int(l % n)
```
Comment (D0048) [inline train_mnist.py:128]: These should be expressed in a clearer manner.

**Review by D0048 — COMMENTED** [review]
By accident I merged this before I could look at it. This merge seems to be a stupid move.

File: train_mnist.py (hunk 1 above)
Comment (ghost) [reply train_mnist.py:134]: It works on my computer.
These change will rise the calculating speed by 70%.

File: train_mnist.py (hunk 2 above)
Comment (ghost) [reply train_mnist.py:128]: That's a stupid thing, I will remove it soon.

File: train_mnist.py (hunk 1 above)
Comment (D0048) [reply train_mnist.py:134]: According the definition here:
 `def small_cnn(x, num_conv, id, j, k, reuse, keep_prob):`, 
I don't see how the call `small_cnn(scn_input, num_conv, keep_prob)` may work by any mean. (And it indeed does not on my computer. )

Comment (ghost) [reply train_mnist.py:134]: It have been fixed in  #5

**Pull request merged**